code/datums/supplypacks.dm

```diff
@@ -1026,7 +1026,7 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 
 /datum/supply_packs/misc/mule
 	name = "MULEbot Crate"
-	contains = list(/obj/machinery/bot/mulebot)
+	contains = list(/mob/living/simple_animal/bot/mulebot)

```

Comment:
Unless someone has changed the supply shuttle, these need to do what the animal crates do and put the animal in after it arrives on station.